client/src/pages/OnBoarding/smallComponents/AvailableHoursButton.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import { TextField, MenuItem } from "@material-ui/core";
+
+const timeSlots = [];
+for (let i = 0; i <= 24; i++) {
+  for (let j = 0; j <= 60; j = j + 15) {
```

Comment:
```suggestion
  for (let j = 0; j < 60; j = j + 15) {
```
This is making times show up as 2:60 for example